parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -228,13 +235,16 @@ public ColumnWriteStore newColumnWriteStore(
           pageStore,
           pageSize,
           dictionaryPageSizeThreshold,
-          enableDictionary, writerVersion);
+          enableDictionary,
+          initialRowCountForSizeCheck,
+          estimateNextSizeCheck,
+          writerVersion);
     case PARQUET_2_0:
       return new ColumnWriteStoreV2(
           schema,
           pageStore,
           pageSize,
-          new ParquetProperties(dictionaryPageSizeThreshold, writerVersion, enableDictionary));
+          new ParquetProperties(dictionaryPageSizeThreshold, writerVersion, enableDictionary, initialRowCountForSizeCheck, estimateNextSizeCheck));
```

Comment:
Why are there only changes to ColumnWriteStoreV1 or ColumnWriterV1? Does V2 not need to be changed?